fhireval/data/__init__.py

```diff
@@ -29,13 +29,135 @@
 from pathlib import Path
 from collections import defaultdict
 import json
+import copy
+from time import sleep 
+import sys
 
-# DO NOT import this. Just use the load_test_data function. It won't reload if the
-# data has already been loaded.
+from pprint import pformat
+from collections import OrderedDict, defaultdict
+
+from fhireval.test_suite import get_identifier
+
+import pytest
+
+import pdb
+
+# DO NOT import any of these. Only use related functions to ensure that client
+# calls are accessing the proper version in case it's replaced during an intialization
+# call
+
+# products from loading JSON objects
+# Please note that once this module's prep_server is run, this will be modified
+# with valid reference information and will not reflect the original source objects
+# completely
 _resource_data = None
 
+# Identifier=>FHIR ID  (identifier is first identifier in the list)
+_inserted_id_list = {}
+
+# Helpful for ensuring that items with dependencies are correctly loaded/deleted
+# in the right order
+_resource_order = [
+    'Organization',
+    'ResearchStudy',
+    'Patient',
+    'Specimen',
+    'ResearchSubject',
+    'Practitioner',
+    'PractitionerRole',

```

Comment:
Shouldn't these come before `ResearchStudy` since `ResearchStudy` refers to `PractitionerRole` which refers to `Practitioner`?

Reply:
Order inside the files is irrelevant. Order is handled by the test module itself.